gradle-jdks/src/main/java/com/palantir/gradle/jdks/PalantirCaPlugin.java

```diff
@@ -34,46 +34,54 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import org.gradle.api.Plugin;
 import org.gradle.api.Project;
 import org.gradle.process.ExecResult;
 
-public final class PalantirCa {
+public final class PalantirCaPlugin implements Plugin<Project> {
     private static final BigInteger PALANTIR_3RD_GEN_SERIAL = new BigInteger("18126334688741185161");
 
-    public static void applyToRootProject(Project rootProject, boolean strict) {
```

Comment:
should `strict` still not be configurable via some extension? the failures will effectively be silent as most builds don't run with `--info` ?

Reply:
We want the failures to be silent, at least on open source, as external contributors who do not have the palantir CA in their truststore need to run the code.

Reply:
Notable: public circleci

Reply:
Internally, it's a different matter - maybe it's good to have this explode if it can't find the cert. However, people shouldn't really be anything other than our internal mirror anyway. We'd probably need to have support for windows if we did do this.

Reply:
Maybe we should warn rather than info internally, so at least it's obvious...

Reply:
It now has configurable log levels, which we can set to warn internally, so people will bug us if this starts to fail at least rather than going unnoticed.